src/docker/compose.dss-local.yaml

```diff
@@ -44,6 +47,21 @@ services:
     volumes:
       - "./processor/config.yaml:/config.yaml"
 
+  ws:
+    build:
+      context: ../../
+      dockerfile: src/docker/api/Dockerfile.ws
+      args:
+        - POSTGRES_WEBSOCKETS_VERSION=0.11.1.0
+    depends_on:
+      - diesel
+      - postgres
+    environment:
+      - PGWS_DB_URI=postgres://econia:econia@postgres/econia
+      - PGWS_JWT_SECRET=econia_is_dope
```

Comment:
Doesn't this mean anyone can just give themselves a valid JWT at any time?
I'm trying to understand the purpose behind having JWT at all given that it seems to provide no security for us.

Reply:
Yes exactly, required by the websockets framework.